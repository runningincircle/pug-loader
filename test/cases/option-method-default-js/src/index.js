// require with alias 'Views'

const tmpl = require('Views/widget.pug');
const html = tmpl({
  title: 'Pug widget',
  text: 'Hello World!',
});

console.log(html);