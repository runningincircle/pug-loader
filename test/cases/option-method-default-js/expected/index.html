<h1>Pug template</h1><h2>Include A</h2><div class="widget test-widget"><span>Pug widget</span><div class="content"><div class="text">Hello World!</div></div></div>